qiskit/assembler/assemble_circuits.py

```diff
@@ -107,13 +107,13 @@ def assemble_circuits(circuits, run_config, qobj_id, qobj_header):
             # to the conditional instruction to map the creg ?= val condition
             # onto a gating register bit.
             if hasattr(instruction, '_control'):
-                ctrl_reg, ctrl_val = instruction._control
+                ctrl = instruction._control
```

Comment:
To me, this seems less readable that what was here prior.

Reply:
undone in 8b66ddbb. It was from a time that I confused bit-tuples with condition-tuples...